srcpkgs/brother-brscan5/template

```diff
@@ -0,0 +1,77 @@
+# Template file for 'brother-brscan5'
+pkgname=brother-brscan5
+version=1.2.12
+revision=1
+archs="i686 x86_64"
+depends="sane libusb-compat"
+short_desc="SANE scanner driver for brscan5-compatible printers"
+maintainer="Alexandru Barbur <alex@ctrlc.name>"
+license="custom:EULA"
+homepage="https://support.brother.com/"
+repository="nonfree"
+nopie=yes
+conf_files="/etc/opt/brother/scanner/brscan5/brscan5.ini
+ /etc/opt/brother/scanner/brscan5/brsanenetdevice.cfg
+ /opt/brother/scanner/brscan5/models/*.ini"
+
+_sane_library="libsane-brother5.so.1.0.7"
+_pkg_libraries="libLxBsScanCoreApi.so.3.2.0
+ libLxBsNetDevAccs.so.1.0.0
+ libLxBsDeviceAccs.so.1.0.0
+ libLxBsUsbDevAccs.so.1.0.0"
+
+# EULA?
+#distfiles="https://support.brother.com/g/s/agreement/English_lpr/agree.html>LICENSE.html"
```

Comment:
These commented lines should be removed.

Reply:
Done, good call.